python/feathub/feature_tables/tests/test_kafka_source_sink.py

```diff
@@ -102,6 +102,8 @@ def test_kafka_source_sink(self):
         )
         self.assertTrue(expected_result_df.equals(result_df))
 
+        self._tear_down_kafka()
```

Comment:
Would it be simpler to teardown `cls.kafka_container` in `KafkaSourceSinkITTest#tearDownClass`?